### ISSUE TITLE
Saved query sidebar tree

### ISSUE THREAD

**Issue by corgus** [jakob/Postico#18]
Love the UI as it is (best I've seen yet!) but have one suggestion (maybe it's more of a dream):
It would be quite useful to have a directory tree similar to Sublime Text's sidebar directory tree, to simplify the 'Load Query' step. Could specify a root folder from Preferences and selecting a .sql file within one of its expandable child directories would load that to the query console...

![screen shot 2015-06-23 at 9 41 03 am](https://cloud.githubusercontent.com/assets/3866184/8311579/02de830a-198c-11e5-913b-699ad53645ed.png)

Not entirely necessary, but would be a pretty great 'nice to have' (:

**Comment by radim** [comment]
+1

**Comment by jakob** [comment]
So I have been working on a feature similar to this. You can try an early build here:
https://eggerapps-downloads.s3-eu-west-1.amazonaws.com/postico-1397.zip

Let me know if you like it, and let me know if you find it confusing. I'm not sure how intuitive this new interface is.

**Comment by corgus** [comment]
@jakob - Great start - some notes on first use:
1) the '+' to create a new query was difficult to find. maybe add this (and new-folder) to the top menu?
2) the 'SQL' within the icon is quite difficult to read
3) would be nice to have ability to reorder queries or group existing queries into folders (or watch directory for updates to sidebar without requiring app restart)
Definitely on the right track!

**Comment by radim** [comment]
the basic usage is definitely or right track. Do you plan any ability to export/import all the "files"? So it can be shared among multiple databases easily (or have that option directly somehow)

**Comment by jakob** [comment]
You can drag files and folders to the sidebar, and Postico will add a link to them. This way you can use any folder on your hard drive to store queries.

You can refresh the sidebar by pressing Cmd-R (future versions will refresh the query sidebar automatically, just haven't implemented that yet)

**Comment by jakob** [comment]
Also, right click on a file and you can show it in Finder. Then you can edit it with other apps, or move it etc. Afterwards, press Cmd-R in Postico to refresh.

**Comment by radim** [comment]
Great. Will try to test it more tonight

**Comment by mritchie712** [comment]
Did this ever get merged to the 1.1.2? It doesn't seem to be there unless I'm missing something 

![image](https://user-images.githubusercontent.com/7364173/28435912-0428ab6e-6d63-11e7-9cff-6658ab632bb7.png)

**Comment by jakob** [comment]
Nope, this branch is still work in progress, and there are still a few major problems I need to fix before I can merge this.
